zrml/market-commons/src/migrations.rs

```diff
@@ -15,3 +15,323 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Zeitgeist. If not, see <https://www.gnu.org/licenses/>.
+
+use crate::{AccountIdOf, BalanceOf, BlockNumberOf, Config, Markets, MomentOf, Pallet};
+use alloc::vec::Vec;
+use core::marker::PhantomData;
+use frame_support::{
+    dispatch::Weight,
+    log,
+    traits::{Get, OnRuntimeUpgrade, StorageVersion},
+    RuntimeDebug,
+};
+use parity_scale_codec::{Decode, Encode};
+use scale_info::TypeInfo;
+use sp_runtime::Perbill;
+use zeitgeist_primitives::types::{
+    BaseAsset, Deadlines, EarlyClose, Market, MarketBonds, MarketCreation, MarketDisputeMechanism,
+    MarketPeriod, MarketStatus, MarketType, OutcomeReport, Report, ScoringRule,
+};
+
+#[cfg(feature = "try-runtime")]
+use crate::MarketIdOf;
+#[cfg(feature = "try-runtime")]
+use alloc::collections::BTreeMap;
+#[cfg(feature = "try-runtime")]
+use alloc::format;
+#[cfg(feature = "try-runtime")]
+use frame_support::migration::storage_key_iter;
+
+#[cfg(any(feature = "try-runtime", feature = "test"))]
+use frame_support::Blake2_128Concat;
+
+#[cfg(any(feature = "try-runtime", test))]
+const MARKET_COMMONS: &[u8] = b"MarketCommons";
+#[cfg(any(feature = "try-runtime", test))]
+const MARKETS: &[u8] = b"Markets";
```

Comment:
Can be grouped in two groups, but irrelevant here as we'll remove that migration in a couple of days again.